torch/fx/graph.py

```diff
@@ -88,6 +88,31 @@ def create_node(self, op: str, target: Target,
         self.nodes.append(n)
         return n
 
+    # sugar for above when you know the op
+    def placeholder(self, name: str) -> Node:
+        return self.create_node('placeholder', name)
+
+    def get_param(self, name: str) -> Node:
+        return self.create_node('get_param', name)
+
+    def call_module(self, 
+                    module_name: str, 
```

Comment:
nit: Maybe name this `module_qualified_name` to emphasize that it's the full qualname?

Also probably a good idea to add docstrings for these methods

Reply:
ill rename it in the next in the stack since this is ready to land